go/vt/discovery/tablet_picker_test.go

```diff
@@ -416,6 +441,33 @@ func TestPickError(t *testing.T) {
 	defer cancel()
 	_, err = tp.PickForStreaming(ctx)
 	require.EqualError(t, err, "context has expired")
+	// if local preference is selected, tp cells include's the local cell's alias
+	require.Greater(t, globalTPStats.noTabletFoundError.Counts()["cell_cella.ks.0.replica"], int64(0))
```

Comment:
And we assume in this case that we then use the alias?

Reply:
yes